ksa_compliance/standard_doctypes/sales_invoice.py

```diff
@@ -1,10 +1,11 @@
 from datetime import date
 from typing import cast
 
-import frappe
-import frappe.utils.background_jobs
 from result import is_ok
 
+import frappe
```

Comment:
These are not sorted properly. Please ensure your configuration matches mine (which I've shared before in a previous PR). Until we get some automated formatting going pre-commit, we have to do it through the IDE and ensure we're using the same configuration.